cmake/std/atdm/waterman/tweaks/CUDA-9.2-RELEASE-CUDA-POWER9-VOLTA70.cmake

```diff
@@ -1,2 +1,11 @@
 INCLUDE("${CMAKE_CURRENT_LIST_DIR}/CUDA_COMMON_TWEAKS.cmake")
 INCLUDE("${CMAKE_CURRENT_LIST_DIR}/ALL_COMMON_TWEAKS.cmake")
+
+#Tests regularaly failing see issue #3939
```

Comment:
@fryeguy52, I think these need to be added to the file `waterman/tweaks/CUDA-9.2-RELEASE_DEBUG-CUDA-POWER9-VOLTA70.cmake`, not this file.

Reply:
@fryeguy52, while we are at it, we should also move the disables:

```
ATDM_SET_ENABLE(PanzerAdaptersSTK_MixedCurlLaplacianExample_DISABLE ON)
ATDM_SET_ENABLE(PanzerAdaptersSTK_MixedCurlLaplacianExample-ConvTest-Tri-Order-1_DISABLE ON)
ATDM_SET_ENABLE(PanzerAdaptersSTK_MixedCurlLaplacianExample-ConvTest-Tri-Order-2_DISABLE ON)
ATDM_SET_ENABLE(PanzerAdaptersSTK_MixedCurlLaplacianExample-ConvTest-Quad-Order-3_DISABLE ON)
ATDM_SET_ENABLE(PanzerAdaptersSTK_CurlLaplacianExample-ConvTest-Quad-Order-2_DISABLE ON)
```

from the file:

* cmake/std/atdm/ATDMDevEnvSettings.cmake

and put them in the file:

* cmake/std/atdm/waterman/tweaks/CUDA-9.2-RELEASE_DEBUG-CUDA-POWER9-VOLTA70.cmake

Does that make sense?

Reply:
@bartlettroscoe yes that makes sense.  I updated the PR to do that as well